test/cmocka/src/math/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@
 
```

Comment:
@keyonjie @singalsu @lgirdwood can this mock test be run directly on the dsp? We want to be able to somehow run some algorithms on the DSP but without really creating an audio pipeline.

So, for example we want to understand how good is our DSP on computing ffts.

Reply:
@dbaluta this might need some logic to run mocks on DSP boot, I think they just run on simulator today. Maybe a Kconfig that calls the Mocks after doing all the init.